js/base/Exchange.js

```diff
@@ -131,7 +131,6 @@ module.exports = class Exchange {
                 'fetchTransfers': undefined,
                 'fetchWithdrawal': undefined,
                 'fetchWithdrawals': undefined,
-                'loadMarkets': true,
```

Comment:
Assuming this removes the parameter from all exchanges - will this then mean that all exchanges are implicitly assumed to support `loadMarkets` and there's no need to check for that before calling this?

Reply:
yes, all exchanges should have loadMarkets as a mandatory base method, and it shouldnt be in `.has` at all. 
however, as loadMarkets call the `fetchMarkets`, i am still waiting to look into `loadMarketsHelper` before this PR will be merged.